apps/src/templates/teacherHomepage/RecentCoursesCollapsible.jsx

```diff
@@ -13,15 +13,16 @@ import shapes from './shapes';
 const RecentCoursesCollapsible = React.createClass({
   propTypes: {
     courses: shapes.courses,
-    showAllCoursesLink: PropTypes.bool.isRequired
+    showAllCoursesLink: PropTypes.bool.isRequired,
+    header: PropTypes.string.isRequired
   },
 
   render() {
-    const { courses, showAllCoursesLink } = this.props;
+    const { courses, showAllCoursesLink, header } = this.props;
 
     return (
       <CollapsibleSection
-        header={i18n.courses()}
+        header={header}
```

Comment:
I usually use header to refer to a big block, and heading or headingText to refer to the text.  I think.